### ISSUE TITLE
PyTables triggers a DeprecationWarning in numpy 1.20: `np.object` is a deprecated alias for the builtin `object`

### ISSUE THREAD

**Issue by JackKelly** [PyTables/PyTables#898]
Here's the full error message :)

```
python3.9/site-packages/tables/array.py:241: 
DeprecationWarning: `np.object` is a deprecated alias
for the builtin `object`. To silence this warning, use `object` by itself. 
Doing this will not modify any behavior and is safe. 
  Deprecated in NumPy 1.20; for more details and guidance:
   https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
    (oid, self.atom, self.shape, self._v_chunkshape) = self._open_array()
```

(Thanks loads for all your work on PyTables, it's super-useful!)

**Comment by avalentino** [comment]
Thanks for reporting

**Comment by wshanks** [comment]
This was fixed in #862 (see this commit: e8d622305ba30762a0ffa3f5b1fcf105a278f255). There are some issues blocking the release of a new version with the fix though: #826.

**Comment by ArchangeGabriel** [comment]
@hadim This one was fixed in https://github.com/PyTables/PyTables/commit/c111bac277e3e34133debcc63c27f480caaae3c6

**Comment by avalentino** [comment]
Fixed in 65af35117b0420d04418a6968456037037d6ea01.

**Comment by simsicon** [comment]
> Fixed in [65af351](https://github.com/PyTables/PyTables/commit/65af35117b0420d04418a6968456037037d6ea01).

Thanks for the fixings, I am wondering if there is a plan to release those fixes, since the version 3.6.1 was released two years ago.
